scripts/deployment/hf_upload/upload.py

```
@@ -0,0 +1,123 @@
+        return False
+    try:
+        result = subprocess.run(['git', 'lfs', 'version'], capture_output=True, text=True, check=True)
+        if result.returncode != 0:
+            logging.warning("Git LFS not available. Install with: git lfs install")
+            return False
+        lfs_patterns = ["*.bin", "*.safetensors", "*.onnx", "*.pkl", "*.pth", "*.pt", "*.h5"]
+        for pattern in lfs_patterns:
+            subprocess.run(['git', 'lfs', 'track', pattern], capture_output=True, text=True, check=True)
+        # Update .gitattributes if exists
+        gitattributes_path = ".gitattributes"
+        if os.path.exists(gitattributes_path):
```

Ruff (0.12.2)

`os.path.exists()` should be replaced by `Path.exists()`

(PTH110)